up.sh

```diff
@@ -0,0 +1,18 @@
+# your local ssh key for connecting to nodes to deploy
+SSH_KEY=${1}
+NODE_OF_NODES=${2:-1}
+# location of node file to upload
+NODE_BIN=${3}
+TWO_MB=$(( 2 * 1024 * 1024 ))
+node_capacity=${1:-$TWO_MB}
+
+
+terraform apply \
+     -var "do_token=${DO_PAT}" \
+     -var "pvt_key=${1}" \
+     -var "number_of_nodes=${NODE_OF_NODES}" \
+     -var "node_bin=${3}"
+
+aws s3 cp ip-list s3://safe-testnet-tool/ip-list --acl public-read
```

Comment:
Noticed that when I cancelled a deploy, i.e. entered `no` at the prompt below:
```
Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value:
```

the script still uploads an empty `ip-list` file to S3 and attempts to register hosts, but obviously none exist.

Can these actions be added under `terraform apply`? Or some other way to only execute when the action is performed?